src/goose.rs

```diff
@@ -134,17 +134,17 @@
 //!     let mut c_task = GooseTask::new(c_task_function);
 //!
 //!     /// A very simple task that simply loads the "a" page.
-//!     fn a_task_function(client: &mut GooseClient) {
+//!     fn a_task_function<'fut>(client: &'fut mut GooseClient) {
```

Comment:
These still use 'fut while the rest used 'r — might be confusing

Reply:
I fixed this as part of getting doc-tests working